packaging/crio.conf.d/00-crio-crun.conf

```diff
@@ -0,0 +1,6 @@
+[crio.runtime.runtimes.crun]
+runtime_path = ""
+runtime_type = "oci"
+runtime_root = "/run/crun"
+runtime_config_path = ""
+monitor_path = ""
```

Comment:
Is there any duplicated configs in the `10-microshift_*.conf` files that could be extracted into this file as well?

Reply:
I won't consider this a blocker for this PR, but it'd be good to follow up on it after this merges.